README.md

```diff
@@ -32,11 +32,17 @@ at the moment.
 
 # 💻 Install and use
 
-In the near future you will be able to find official releases
-in Github, PyPi and through AWS. Until then the easiest way
-to install and use Grants tagger is through a
-manual installation. You need to git clone and pip install
-the code in your environment of choice.
+Easiest way to install is with pip.
+The project is available in the PyPI repository at https://pypi.org/project/grants-tagger/.
+
+```
+pip install grants-tagger
+```
+
+Alternatively you can clone this repository and build the project with 
```

Comment:
This would not install the project, this would create the development environment for which we give instructions later. The manual installation would be to clone the project and run `pip install setup.py`